package.json

```diff
@@ -13,30 +13,27 @@
   "license": "Apache-2.0",
   "author": "Google LLC",
   "bin": {
-    "gapic-generator-typescript": "build/src/start-script.js",
-    "protoc-gen-typescript_gapic": "build/src/cli.js"
+    "gapic-generator-typescript": "build/src/gapic-generator-typescript.js",
+    "protoc-gen-typescript_gapic": "build/src/protoc-plugin.js"
   },
   "files": [
-    "build/src",
-    "pbjs-genfiles",
-    "templates"
+    "build/**/*.js",
+    "build/**/*.d.ts",
+    "build/templates"
   ],
   "scripts": {
-    "baseline": "node build/tools/update-baselines.js",
-    "clean": "gts clean",
-    "codecov": "c8 --reporter=lcov mocha build/test/unit && c8 report",
-    "compile": "tsc -p .",
-    "compile-protos": "pbjs -p protos -p node_modules/google-gax/protos -t static-module -o pbjs-genfiles/plugin.js google/protobuf/compiler/plugin.proto google/api/annotations.proto google/api/field_behavior.proto google/api/resource.proto google/api/client.proto google/longrunning/operations.proto service_config.proto && pbts pbjs-genfiles/plugin.js -o pbjs-genfiles/plugin.d.ts",
+    "baseline": "node bazel-bin/tools/update-baselines.js",
+    "clean": "bazel clean && rm -rf build",
+    "codecov": "c8 --reporter=lcov mocha bazel-bin/test/unit && c8 report",
+    "compile": "bazel build //...",
     "docker-test": "sh docker/test.sh",
-    "fix": "gts fix",
-    "js-test-application": "mocha build/test/test-application/test-js --timeout 600000",
-    "lint": "gts check",
-    "prepack": "cd templates/typescript_gapic && rm -f package.json.njk && mv package.json package.json.njk",
+    "fix": "gts fix src/**/*.ts test/**/*.ts tools/*.ts",
```

Comment:
Why does this need to be overidden?

Reply:
I wasn't able to make `eslint` work but now I've got an idea - maybe it just started processing `bazel-*` folders?  I'll try fixing it now.

Reply:
Fixed!